test/src/test/java/org/corfudb/integration/LogReplicationAbstractIT.java

```diff
@@ -0,0 +1,362 @@
+        }
+    }
+
+    public void stopActiveLogReplicator() {
+        List<String> paramsPs = Arrays.asList("/bin/sh", "-c", "ps aux | grep CorfuInterClusterReplicationServer | grep 9010");
+        String result = runCommandForOutput(paramsPs);
+
+        // Get PID
+        String[] output = result.split(" ");
+        int i = 0;
+        String pid = "";
+        for (String st : output) {
+            if (!st.equals("")) {
+                i++;
+                if (i == 2) {
+                    pid = st;
+                    break;
+                }
+            }
+        }
+
+        System.out.println("*** Active PID :: " + pid);
+
+        List<String> paramsKill = Arrays.asList("/bin/sh", "-c", "kill -9 " + pid);
+        runCommandForOutput(paramsKill);
+
+        if (activeReplicationServer != null) {
+            activeReplicationServer.destroyForcibly();
+            activeReplicationServer = null;
+        }
+    }
+
+    public static String runCommandForOutput(List<String> params) {
+        ProcessBuilder pb = new ProcessBuilder(params);
+        Process p;
+        String result = "";
+        try {
+            p = pb.start();
+            final BufferedReader reader = new BufferedReader(new InputStreamReader(p.getInputStream()));
+
+            StringJoiner sj = new StringJoiner(System.getProperty("line.separator"));
+            reader.lines().iterator().forEachRemaining(sj::add);
+            result = sj.toString();
+
+            p.waitFor();
+            p.destroy();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
+        return result;
+    }
+
+    public void startActiveLogReplicator() {
+        try {
+            if (runProcess) {
+                // Start Log Replication Server on Active Site
+                activeReplicationServer = runReplicationServer(activeReplicationServerPort, pluginConfigFilePath);
+            } else {
+                executorService.submit(() -> {
+                    CorfuInterClusterReplicationServer.main(new String[]{"-m", "--plugin=" + pluginConfigFilePath, "--address=localhost", String.valueOf(activeReplicationServerPort)});
+                });
+            }
+        } catch (Exception e) {
+            System.out.println("Error caught while running Log Replication Server");
+        }
+    }
+
+    public void verifyDataOnStandby(int expectedConsecutiveWrites) {
+        // Wait until data is fully replicated
+        while (mapAStandby.size() != expectedConsecutiveWrites) {
+            // Block until expected number of entries is reached
+        }
+
+        // Verify data is present in Standby Site
+        assertThat(mapAStandby.size()).isEqualTo(expectedConsecutiveWrites);
+
+        for (int i = 0; i < (expectedConsecutiveWrites); i++) {
+            assertThat(mapAStandby.containsKey(String.valueOf(i)));
+        }
+    }
+
+    public void checkpoint(CorfuRuntime rt, List<CorfuTable> toCheckpoint) {
```

Comment:
can we rename this also to checkPointAndTrim?

Reply:
Sure. Done.